src/annotator/integrations/index.js

```diff
@@ -16,7 +21,14 @@ import { PDFIntegration, isPDF } from './pdf';
 export function createIntegration(annotator) {
   if (isPDF()) {
     return new PDFIntegration(annotator);
-  } else {
-    return new HTMLIntegration();
   }
+
+  const vsFrameRole = vitalSourceFrameRole();
+  if (vsFrameRole === 'container') {
+    return new VitalSourceContainerIntegration(annotator);
+  } else if (vsFrameRole === 'content') {
+    return new VitalSourceContentIntegration();
+  }
```

Comment:
A couple of thoughts:

As we add more integrations (content types, as it were), we may get to a point where we need to have a think about precedence. For now, this all reads quite easily.

The little, childish part of my brain that wants perfect symmetry and balance wishes that the VitalSource integration could be "one thing" instead of two (content vs. container), but my more grown-up brain recognizes that that would probably introduce an extra layer of potentially-stupid abstraction. The way you've structured `vitalSourceFrameRole` helps here. I wonder if integration-"sniffing" functions (`isPDF`, `vitalSourceFrameRole`) could have some sort of naming convention...

Reply:
I didn't try to hard to standardize the interface for creating integrations yet because I'm not sure what the appropriate interface will be. I figure that will become clearer as requirements evolve.